ConnectorNavisworks/ConnectorNavisworks.2020/ConnectorNavisworks.2020.csproj

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Finessing, but I think other connectors don't have a period in the project name to separate the version number ;)